### PR TITLE
Rewrite of german Credit Part 1

### PR DESCRIPTION
**Pull request by pkopper**
Basically I added a few things but mostly I improved the readability to "real text"
The last section, however, is entirely new.

### CODE REVIEW

**Review by pat-s — APPROVED** [review]

File: content/post/mlr3_basics_german_credit.Rmd [hunk 1]
```diff
@@ -72,8 +69,13 @@ theme_set(theme_light())
...
 
-- The German credit data is a research data set of the University of Hamburg from 1994 donated by Prof. Hans Hoffman.
-- Description (and manual download) can be found at the [UCI repository](https://archive.ics.uci.edu/ml/datasets/Statlog+%28German+Credit+Data%29).
+In our tutorials we use the `german_credit` dataset for our next example.
+The German credit data is a research data set of the University of Hamburg from 1994 donated by Prof. Hans Hoffman.
+The data features personal, demographic and financial data on individuals.
+Furthermore, it contains an assessment on the individual's creditworthniness.
+A detailed description (and manual download) can be found at the [UCI repository](https://archive.ics.uci.edu/ml/datasets/Statlog+%28German+Credit+Data%29).
+
+As the `"GermanCredit"` data set is frequently used within `r mlr_pkg("mlr3")`, there is a task already existing just for this purpose.
```
Comment (pat-s) [inline content/post/mlr3_basics_german_credit.Rmd:78]: `mlr_pkg()` only works in the mlr3book.

**Review by pat-s — CHANGES_REQUESTED** [review]
Thanks! Please fix the CI build.

File: content/post/mlr3_basics_german_credit.Rmd [hunk 2]
```diff
@@ -113,14 +115,19 @@ credit = task$data()
 ```
 
 ### Exploring the Data
-- We have a look at the data set before we start modeling.
```
Comment (pfistfl) [inline content/post/mlr3_basics_german_credit.Rmd:116]: Why remove this?

**Comment by pat-s** [comment]
@pkopper Please add a short notice why you close a PR like this suddenly.

**Pull request closed without merging**